### DISCUSSION TITLE
machine-learning/linear-model/regularization

### DISCUSSION THREAD

**Discussion by giscus** [luweizheng/lulaoshi.info#73] [Announcements]
# machine-learning/linear-model/regularization

正则化可以有效防止模型过拟合。

https://lulaoshi.info/machine-learning/linear-model/regularization.html

**Comment by crush598** [comment]
鲁老师好，岭回归的第一个公式可能没有正确显示出来？

**Reply by luweizheng** [reply]
> 鲁老师好，岭回归的第一个公式可能没有正确显示出来？

谢谢提醒，已修复。
